rholang/src/test/scala/coop/rchain/rholang/interpreter/util/AddressToolsSpec.scala

```diff
@@ -0,0 +1,59 @@
+package coop.rchain.rholang.interpreter.util
+import coop.rchain.crypto.hash.Blake2b256
+import org.scalacheck._
+import org.scalatest.prop.GeneratorDrivenPropertyChecks
+import org.scalatest.{FlatSpec, Matchers, PropSpec}
+import cats.implicits._
+import coop.rchain.rholang.interpreter.util.codec.Base58
+
+class AddressToolsSpec extends PropSpec with GeneratorDrivenPropertyChecks with Matchers {
```

Comment:
we should probably bump the property check config to do more trials of bigger size

Reply:
done